sdk/identity/Azure.Identity.Broker/api/Azure.Identity.Broker.net462.cs

```diff
@@ -9,6 +9,6 @@ public partial class SharedTokenCacheCredentialBrokerOptions : Azure.Identity.Sh
     {
         public SharedTokenCacheCredentialBrokerOptions() { }
         public SharedTokenCacheCredentialBrokerOptions(Azure.Identity.TokenCachePersistenceOptions tokenCacheOptions) { }
-        public bool? IsMsaPassthroughEnabled { get { throw null; } set { } }
+        public bool? IsLegacyMsaPassthroughEnabled { get { throw null; } set { } }
```

Comment:
Why is this line changed?

Reply:
Only this line changes, therefore it is none of my business - they must have this change but do not have its api generated accordingly

Reply:
Code here: https://github.com/azure-sdk/azure-sdk-for-net/blob/fa382c90f3fba485f8061fe98e6ef7b00370fd56/sdk/identity/Azure.Identity.Broker/src/SharedTokenCacheCredentialBrokerOptions.cs#L19C22-L19C52